src/pyasm/search/upgrade/project/config_upgrade.py

```diff
@@ -19,6 +19,27 @@
 class ConfigUpgrade(BaseUpgrade):
 
 
+    #
+    # 4.7.0.a01
+    #
+    def upgrade_v4_7_0_a01_001(self):
+        self.run_sql('''
+        CREATE TABLE "spt_process_state" (
+            "id" serial PRIMARY KEY,
+            "code" character varying(256),
+            "pipeline_code" character varying(256),
+            "process" character varying(256),
+            "search_type" character varying(256),
+            "search_code" character varying(256),
+            "timestamp" timestamp without time zone DEFAULT now() NOT NULL,
```

Comment:
timestamp without timezone may not exist in all database types

Reply:
Added MySQL section with "timestamp DEFAULT CURRENT_TIMESTAMP NOT NULL"